.github/ISSUE_TEMPLATE/design-a-component.md

```diff
@@ -0,0 +1,44 @@
+---
+name: "Design a component"
+about: Everything we need to do when we design a component
+title: ''
+labels: "contribution or major iteration, design, epic"
+assignees: ''
+
+---
+
+## What
+<!-- Add name of component, plus any relevant links (for example, community backlog issues and comments, pull requests for the component and its guidance) -->
+
+## Why
+<!-- Add reason for designing the component -->
+
+## Who needs to know about this
+<!-- Add team roles involved in designing the component -->
+
+## Acceptance criteria
+<!-- Customise, and add component-specific checklist items to, this list of criteria all components need to meet -->
+### Design
+The component must:
+- [ ] be visually consistent with GOV.UK
+- [ ] [not rely on colour to communicate information](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [ ] use the GOV.UK colour palette
+- [ ] not use icons
+
+### Guidance
+The component's guidance must include these sections:
+- [ ] 'When to use this component' (use cases)
+- [ ] 'When not to use this component' (use cases for other components)
+- [ ] 'How it works' (implementation guidance )
+- [ ] 'Research on this component' (further research that's required)
+- [ ] 'Help improve this page' (Github links to discussion of component and how to propose changes)
+
+### Usability
+The component must:
+- [ ] have a clear use case
+- [ ] be usable in all government services
+- [ ] be flexible enough for use in multiple use cases
+- [ ] be in use in at least 3 government services
```

Comment:
Could we involve the community here? To ask for:

- examples
- evidence from user testing
- iterations made
- rationale behind changes